esdoc.json

```diff
@@ -38,6 +38,10 @@
       "./doc/tutorial/basics.md",
       "./doc/tutorial/applications.md",
       "./doc/external-references.md"
+    ],
+    "usage": [
+      "./doc/writing-marble-tests.md",
+      "./doc/operator-creation.md"
```

Comment:
The "usage" section may be misleading to beginners because typically (e.g. in readme files) the usage section explains how to use the library after installing, but in this case "usage" is just a place for advanced content. I would recommend placing it under Tutorial somewhere, instead.